dds/DCPS/Time_Helper.inl

```diff
@@ -0,0 +1,245 @@
+/*
+ *
+ *
+ * Distributed under the OpenDDS License.
+ * See: http://www.opendds.org/license.html
+ */
+
+#include "ace/OS_NS_string.h"
+#include "ace/Truncate.h"
+
+#include <cstring>
+
+namespace {
+
+const ACE_UINT32 NSECS_IN_SEC = 1000000000;
```

Comment:
I realize this was the same in the old file, but I'd rather not have anonymous-namespace objects in .inl files.  Since the constant is only used in one function we could have it as a static const local to that function.